docs/deploy.md

```diff
@@ -155,7 +155,10 @@ WorkerRef: null
 ```
 
 ## Deploy Spark client
-Copy `$CELEBORN_HOME/spark/*.jar` to `$SPARK_HOME/jars/`.
+Celeborn release binary contains clients for Spark 2.x and Spark 3.x, copy the corresponding client jar into Spark's
+`jars/` directory:
+
+Copy `$CELEBORN_HOME/spark/<Celeborn Client Jar>` to `$SPARK_HOME/jars/`.
```

Comment:
The `<Celeborn Client Jar>` is also ambiguous.

Reply:
What about <celeborn-client-spark-{version} Jar>

Reply:
LGTM.

Reply:
@zaynt4606, `celeborn-client-spark-<spark.major.version>-shaded_<scala.binary.version>-<celeborn.version>.jar`, WDYT?